Samples/native/example/OneSignalWidget/OneSignalWidget.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  OneSignalWidget.swift
+//  OneSignalWidget
+//
+//  Created by Brian Smith on 11/30/22.
+//
+
+import WidgetKit
```

Comment:
This file is unused. This is for a (non-live activity) widget.

Reply:
Yup!  I kept that in there to help through the setup process as it was easier to get the widget showing than the live activity.  It is included in the widget bundle for this purpose.